app/views/admin/editions/_datetime_fields.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="gem-c-date-input govuk-date-input govuk-datetime <%= 'govuk-form-group--error' if errors_for_input(edition.errors, field_name).present? %>">
+  <h3 class="govuk-heading-s govuk-!-margin-bottom-2 govuk-!-margin-top-4">Date</h3>
+
+  <div class="gem-c-hint govuk-hint govuk-!-margin-bottom-2">
+    For example, 01 August 2022
+  </div>
+
+  <% if errors_for_input(edition.errors, field_name).present? %>
+    <p class="gem-c-error-message govuk-error-message">
+      <span class="govuk-visually-hidden">Error:</span> <%= errors_for_input(edition.errors, field_name) %>
+    </p>
+  <% end %>
+
+  <div class="govuk-date-input__item">
+    <div class="govuk-form-group">
+      <label for="edition_<%= field_name %>_3i" class="gem-c-label govuk-label">Day</label>
+      <%= select_day edition.send(field_name)&.day, { include_blank: true, prefix: "edition", field_name: "#{field_name}(3i)" } %>
```

Comment:
### Opinion

I think it's usually preferable to use `public_send` rather than `send` in situations like this – since we should only be calling public methods/attributes on the model from here anyway.

```suggestion
      <%= select_day edition.public_send(field_name)&.day, { include_blank: true, prefix: "edition", field_name: "#{field_name}(3i)" } %>
```

(same applies to the other inputs in this partial)